packages/cli/src/WebhookHelpers.ts

```diff
@@ -213,8 +217,22 @@ export async function executeWebhook(
 		throw new ResponseHelper.ResponseError(errorMessage, 500, 500);
 	}
 
+	let user: N8nUserData;
```

Comment:
Not clear on why `N8nUserData` and not `User`. The relation will return a `User`. 

Also, if `N8nUserData` is meant to be the sanitized version, we might consider renaming `N8nUserData` to something more explicit. Didn't we use to have `PublicUserData`?

Reply:
It is not a sanitized version, actually we need a representation of the user residing in the `workflow` package.

Similar to what we have as `packages/cli/src/databases/entities/WorkflowEntity.ts` that is a database entity representing a workflow and `packages/workflow/src/Workflow.ts` which is a similar object (contains the same properties but more methods).

Perhaps it would be better to follow this pattern and rename:
- `User` entity in the `cli` package becomes `UserEntity`
- `N8nUserData` in the `workflow` package becomes `User`

We should discuss this with @BHesseldieck.